code/game/machinery/cloning.dm

```diff
@@ -373,16 +378,19 @@
 		to_chat(occupant, "<span class='warning'><b>Agony blazes across your consciousness as your body is torn apart.</b><br><i>Is this what dying is like? Yes it is.</i></span>")
 		playsound(src.loc, 'sound/machines/warning-buzzer.ogg', 50, 0)
 		occupant << sound('sound/hallucinations/veryfar_noise.ogg',0,1,50)
-		QDEL_IN(occupant, 40)
+		var/obj/item/organ/brain/B = occupant.getorganslot("brain")
+		B.Remove(occupant)
+		B.forceMove(src)
+		occupant.gib(TRUE, TRUE, TRUE)

```

Comment:
Calling qdel on a mob does not leave them in null space, and emagging the cloner isn't supposed to leave any clonable remains.